app.py

```diff
@@ -219,3 +220,52 @@ def list_project():
             "text": message,
         }
     )
+
+def fetch_github_data(owner, repo):
+    prs = requests.get(f'{GITHUB_API_URL}/repos/{owner}/{repo}/pulls?state=closed').json()
+    issues = requests.get(f'{GITHUB_API_URL}/repos/{owner}/{repo}/issues?state=closed').json()
+    comments = requests.get(f'{GITHUB_API_URL}/repos/{owner}/{repo}/issues/comments').json()
+    
+    return prs, issues, comments
+
+def format_data(prs, issues, comments):
+    user_data = {}
+
+    for pr in prs:
+        user = pr['user']['login']
+        if user not in user_data:
+            user_data[user] = {'prs': 0, 'issues': 0, 'comments': 0}
+        user_data[user]['prs'] += 1
+
+    for issue in issues:
+        user = issue['user']['login']
+        if user not in user_data:
+            user_data[user] = {'prs': 0, 'issues': 0, 'comments': 0}
+        user_data[user]['issues'] += 1
+
+    for comment in comments:
+        user = comment['user']['login']
+        if user not in user_data:
+            user_data[user] = {'prs': 0, 'issues': 0, 'comments': 0}
+        user_data[user]['comments'] += 1
+
+    table = "User | PRs Merged | Issues Resolved | Comments\n ---- | ---------- | --------------- | --------\n"
+    for user, counts in user_data.items():
+        table += f"{user} | {counts['prs']} | {counts['issues']} | {counts['comments']}\n"
```

Comment:
I suggest using list and append rows to it. Then you can join it with `\n`. Don't use string concatenation.